docs/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  rules: {
+    'react/prop-types': 'off',
+    'react/react-in-jsx-scope': 'off',
+    'react/no-unknown-property': ['error', { ignore: ['sx'] }],
+    'import/extensions': [
+      'error',
```

Comment:
Config needs to move to the root folder eslint config file to prepare Eslint v9 upgrade.